shell/browser/native_window_mac.mm

```diff
@@ -291,7 +291,19 @@ static bool FromV8(v8::Isolate* isolate,
 
   // Resize to content bounds.
   bool use_content_size = false;
+  int inner_width = 0;
+  int inner_height = 0;
   options.Get(options::kUseContentSize, &use_content_size);
+  options.Get(options::kinnerWidth, &inner_width);
+  options.Get(options::kinnerHeight, &inner_height);
+  if (inner_width || inner_height) {
+    use_content_size = true;
+    if (inner_width)
+      width = (inner_width < 100) ? 100 : inner_width;
+    if (inner_height)
+      height = (inner_height < 100) ? 100 : inner_height;
```

Comment:
1. Better to use `foo = std::max(a, inner_foo)` here

2. We're still allowing a value of <100 to be inherited from either `width` or `height` via `options::kWidth`. Is that OK?

Reply:
It's also not clear to me _why_ 100 is being used here?

Reply:
@codebytere it's in the spec:

https://developer.mozilla.org/en-US/docs/Web/API/Window/open#width

> Specifies the width of the content area, including scrollbars. The minimum required value is 100.

But you're right, that's  not clear unless one reads the spec or this thread. Maybe a symbolic constant name could be used to make it clearer, e.g.

```objc++
constexpr int kMinSizeReqdBySpec = 100;
```

Reply:
ah cool - ty!

Reply:
I think since this is supposed to be inline with the window.open spec, our docs do not need to also state that it adheres to this requirement specifically. Although, I think linking to the spec in the code is helpful to understand what might seem like an arbitrary value at first glance.